quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/WildcardPropertyMapper.java

```diff
@@ -42,6 +42,10 @@ public WildcardPropertyMapper(Option<T> option, String to, BooleanSupplier enabl
             throw new IllegalArgumentException("Invalid wildcard from format. Wildcard must be at the end of the option.");
         }
 
+        if (option == LoggingOptions.LOG_LEVEL_CATEGORY) {
+            replacementChar = '.';
```

Comment:
jfyi - It seems fine to have it hardcoded for now, and if we have some other requirements for replacement char, we can adjust it. So, +1 to have it like this.